src/sass/components/_tour.scss

```diff
@@ -34,3 +34,10 @@
 .slds-popover_walkthrough .slds-popover__header {
   background-image: url('~img/popover-header.png');
 }
+
+// Keep text from being truncated in table and page header
+.slds-popover {
+  font-weight: normal;
+  text-align: left;
+  white-space: initial;
+}
```

Comment:
Oh, huh. Can this replace the `isAlert` classes we're using in the `dropdown` component?

Reply:
Was able to replace everything but kept the one that sets the font-color to default for now. If we get rid of it I think we would have to make a new class with a hard-coded color?